src/main/java/io/stargate/sgv2/jsonapi/service/operation/model/ReadOperation.java

```diff
@@ -363,6 +393,17 @@ default Uni<CountResponse> countDocuments(
             });
   }
 
+  private void getCount(AsyncResultSet rs, Throwable error, AtomicLong counter) {
+    if (error != null) {
+      throw new RuntimeException(error);
```

Comment:
Would it be possible/better to throw JsonApiException from here, with extra information we may have about count query? (or at least indicate context as failure of COUNT query, with separate `ErrorCode`? Could indicate how many rows have been counted at minimum; page size used etc, making debugging of failures easier)